src/pages/PersonalData.test.js

```diff
@@ -0,0 +1,38 @@
+import "@testing-library/jest-dom";
+import "@testing-library/user-event";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event/";
+import PersonalData from "./PersonalData";
+
+describe("PersonalData", () => {
+  test("check the validation of form", async () => {
+    render(<PersonalData />);
+
+    const form = screen.getByTestId("form");
+    const submitbtn = screen.getByRole("button", { name: "Submit" });
+
+    const getNameInputfield = screen.getByRole("textbox", { name: /email/i });
+    const getEmailInputfield = screen.getByRole("textbox", { name: /name/i });
+
+    await userEvent.type(getNameInputfield, "Nazar");
+    await userEvent.type(getEmailInputfield, "jjj@sd.sd");
+
+    expect(form).toBeInTheDocument();
+    expect(submitbtn).toBeInTheDocument();
+
+    expect(getNameInputfield).toBeInTheDocument();
+    expect(getEmailInputfield).toBeInTheDocument();
```

Comment:
You did not check, if the typed value was really typed. You just checking if the element is present.
If I remove the `@` from the `getEmailInputfield` type event, the test still passes. This is not how "validation" should be working. This test needs rework.